drivers/hmis/app/graphql/types/hmis_schema/ce_client.rb

```diff
@@ -0,0 +1,90 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+module Types
+  class HmisSchema::CeClient < Types::BaseObject
+    description 'A client who is a candidate for Coordinated Entry (CE), represented by a ClientProxy. Underlying client record is Destination Client.'
```

Comment:
```suggestion
    description 'A client who is eligible for Coordinated Entry (CE), represented by a ClientProxy. Underlying client record is Destination Client.'
```
Suggest removing "candidate" language here, to more clearly differentiate `CeClient`s from `CeCandidate`s. Also, maybe we could add more comments to either/both graphql objects to help clarify the distinction. Here's my understanding, let me know if it sounds right:
- `CeCandidate` - represents a person's eligibility _for a particular candidate pool_, with a priority score representing their spot on the waitlist for that pool. In general, this object won't be resolved unless there is an open opportunity for this pool(?)
- `CeClient` -> represents a person who is eligible for at least one candidate pool, regardless of availability.

Reply:
Yep that sounds right to me! I will update the description.